packages/ant/build.sh

```diff
@@ -5,22 +5,16 @@ TERMUX_PKG_SRCURL=http://apache.mirrors.spacedump.net//ant/binaries/apache-ant-$
 TERMUX_PKG_SHA256=ffc6c9eb2d153db76e18df70fa145744ddd0d754339df5a797a47603cfeaa8a6
```

Comment:
Package modifications require revision bump.

Reply:
Whoops, that didn't come into mind.